.jenkins/cscs/config-gcc-cuda-11-stdexec.sh

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2020 ETH Zurich
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+configuration_slurm_constraint="gpu"
+configuration_skip_pr="true"
```

Comment:
You might want to set
```suggestion
configuration_skip_pr="false"
```
for now if you want to test this in CI. In this case it may be easier to just test it manually though.